Library/Homebrew/extend/hash.rb

```diff
@@ -0,0 +1,11 @@
+class Hash
```

Comment:
Why? Current PR (a) makes code much easy to read and use from OO respective, (b) it will has its own build-in type check.